### DISCUSSION TITLE
Is it possible to save all recordings, motion recordings and tracked object recordings for different durations?

### DISCUSSION THREAD

**Discussion by naskinner999** [blakeblackshear/frigate#5045] [Ask A Question] [answered]
Hi,
I am pretty new to Frigate and have it running through the Home Assistant add on.
I have it configured to save motion recordings for a number of days, and tracked objects for a longer period of time. This is configured on a per camera basis.
This is working well.
I had hoped, in addition to this, to save all recordings for a determined period of time, e.g. All for 1 day, motion for 3 days, tracked objects for 7 days.
I have attempted to configure recording all globally, hoping that the individual camera settings wouldn't override it, but the behaviour hasn't changed.
Is what I'm hoping to achieve possible, and if so, how do I achieve it..?

Thanks in anticipation.

Current config (only one camera included, all others have similar configuration):

`mqtt:
  host: XXX.XXX.XX.XXX
  user: XXXXXXX
  password: XXXXXXX
ffmpeg:
  output_args:
    #record: -f segment -segment_time 10 -segment_format mp4 -reset_timestamps 1 -strftime 1 -c copy -an  # Disable sound
    record: -f segment -segment_time 10 -segment_format mp4 -reset_timestamps 1 -strftime 1 -c:v copy -c:a aac # Enable sound
  
  hwaccel_args: -hwaccel vaapi -hwaccel_device /dev/dri/renderD128 -hwaccel_output_format yuv420p
detectors:
  cpu:
    type: cpu

record:
  retain:
    days: 1
    mode: all

cameras:
  FrontLeft: 
    ui:
      order: 1
    ffmpeg:
      inputs:
        - path: rtsp://XXXXXXXXXXX # High Resolution Stream
          roles:
            - record
            - rtmp
        - path: rtsp://XXXXXXXXXXXX # Low Resolution Stream
          roles:
            - detect
    rtmp:
      enabled: True 
    detect:
      width: 1280 
      height: 720 
      fps: 5
    objects:
        track:
          - person
    snapshots:
      enabled: True
    record:
      enabled: True
      retain:
        days: 5
        mode: motion
      events:
        retain:
          default: 14
          mode: active_objects
    motion:
      mask:
        - 1273,0,1280,168,373,72,0,157,260,388,609,594,439,720,0,720,0,0`

**Comment by NickM-27** [comment, marked as answer]
No this is not currently possible. There's an open pinned feature request for this.
